projects/demo-playwright/tests/kit/combo-box/combobox.spec.ts

```diff
@@ -24,4 +26,122 @@ describe(`ComboBox`, () => {
         await input.click();
         await expect(page).toHaveScreenshot(`01-combobox-dont-allow-disabled-04.png`);
     });
+
+    describe(`API`, () => {
+        [true, false].forEach(strict => {
+            test(`search shouldn't be reset if an exact match is entered when strict is ${strict}`, async ({
+                page,
+            }) => {
+                const {apiPageExample} = new TuiDocumentationPagePO(page);
+
+                const {comboboxTextfield} = new TuiComboBoxPO(apiPageExample);
+
+                await visitBy(page, strict);
+                await openFormValue(apiPageExample);
+                await expect(page).toHaveScreenshot(
+                    `search-should-not-be-reset-strict-${strict}.png`,
```

Comment:
Too large snapshot

![search-should-not-be-reset-strict-true-chromium-darwin](https://github.com/taiga-family/taiga-ui/assets/35179038/32367a15-a1f6-446d-8123-ed4ec3dfaf24)

(+ other snapshots have the same problem)

Reply:
@nsbarsukov friendly ping

Reply:
> May I just reduce viewport?

Just use `setViewportSize`, yep

Reply:
\+ rebase the branch, please (it contains merge conflicts with `main` 😢 )